File: Module2/src/test/student_manager_compare/model/Employee.java
package test.student_manager_compare.model;

import test.student_manager_compare.model.Person;

public class Employee extends Person {
    int Salary;
    public Employee(){

    }
    public Employee(int salary) {
        Salary = salary;
    }

    public Employee(int id, String name, String address, int salary) {
        super(id, name, address);
        Salary = salary;
    }

    public int getSalary() {
        return Salary;
    }

    public void setSalary(int salary) {
        Salary = salary;
    }
}

File: Module5/caseStudy01/src/app/models/Employee.ts
export class Employee {
  id: number;
  name: string;
  dateOfBirth: string;
  address: string;


  constructor(id: number, name: string, dateOfBirth: string, address: string) {
    this.id = id;
    this.name = name;
    this.dateOfBirth = dateOfBirth;
    this.address = address;
  }
}
